documentation/man/features/kw-drm.rst

```diff
@@ -39,7 +39,7 @@ OPTIONS
   will make sure that the target module will be load first and after that
   trigger the GUI.
 
--um=<module>[;<module>;...], \--unload-module=<module>[;<module>;...]:
+-um, \--unload-module=<module>[;<module>;...]:
```

Comment:
This is the issue that I was talking about in another comment.

https://github.com/kworkflow/kworkflow/issues/1002

Could you address it here? I think we can just drop -um and -lm.

Reply:
The issue #1002 was already included in the commit `src: plugins: subsystems: drm: add returns to [un]load_module functions`, which is more related to the issue.